src/js/saspowerschoolff.js

```diff
@@ -143,14 +144,15 @@ function main_page () {
                         courses_first_semester.push({
                             name: $prev_course.getElementsByTagName("td")[0].textContent.trim(),
                             grade: $prev_course.getElementsByTagName("td")[1].textContent.trim(),
-                            link: $prev_course.getElementsByTagName("td")[2].getElementsByTagName("a")[0].href,
                             fp: -1,
 
                         });
                     }
                     $("table[border='0'][cellpadding='3'][cellspacing='1'][width='100%']").prepend(`<tr><td align="center">Last Semester GPA (S1): ${calculate_gpa(courses_first_semester)}</td></tr>`);
                 }
             });
+        $("table[border='0'][cellpadding='3'][cellspacing='1'][width='100%']").prepend(`<td style="background-color: white;" align="center"><button id="calculateCumulative">Calculate Cumulative GPA</button></td>`);
+        $("#calculateCumulative").click(function () { show_cumulative_gpa(); });
```

Comment:
```suggestion
        $("#calculateCumulative").click(show_cumulative_gpa);
```
No need to make a function just to call another function :).